Fix quizzes

Felienne opened the pull request:
Fixes #1387

I have simply split the quiz for the old level 2 into two parts, and made some other small fixes. We do need a few more questions in levels 2 and 3 but at least it is not incorrect anymore. Would be good if someone could do a check if I did not miss anything important.

boryanagoncharenko commented:
I checked the content of the levels in English and fixed small typos. Also, addressed 2 validation issues: the values of correct_answer were not valid in 2 places. I did not check the Dutch quizzes, so probably you want someone else to have a look at them.

In this PR https://github.com/Felienne/hedy/pull/1415 I consolidate the quiz levels in language files (so that they are aligned with the rest of the yaml files in Hedy). Once this is approved, I will update the content there.

Felienne commented:
> I checked the content of the levels in English and fixed small typos. Also, addressed 2 validation issues: the values of correct_answer were not valid in 2 places. I did not check the Dutch quizzes, so probably you want someone else to have a look at them.
> 
> In this PR #1415 I consolidate the quiz levels in language files (so that they are aligned with the rest of the yaml files in Hedy). Once this is approved, I will update the content there.

And thanks for thinking of this one, I was worried it would get mixed up.
 
I am not sure of the order though, do you want me to first merge this and then #1415 or the other way around?

boryanagoncharenko commented:
Let's merge this one first. Updating the other one will be easy.
